src/test/java/HW/HW3_1.java

```diff
@@ -0,0 +1,185 @@
+package HW;
+
+import Class1.practice;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+
+public class HW3_1 {
+
+    @Test
+    public void CorrectTemp() {
+        practice.openURL("https://www.darksky.net/");
+        String taketempnow = "//span[@class='summary swap']";
+        By tempnowLocator = By.xpath(taketempnow);
+        WebElement tempnowElement = practice.getDriver().findElement(tempnowLocator);
+        String curTempfar = tempnowElement.getText();
+
+        String currentTempFar = "//div[@id='header']/div[1]/div[1]/div[2]/span[1]";
```

Comment:
This xpath got too many dependencies. Its not good bcz if any div get changed then xpath not gonna work. So try to create dynamic xpath.